package.json

```diff
@@ -63,7 +63,8 @@
     "prettier": "^2.7.1",
     "probot": "^12.2.4",
     "tsx": "^3.12.7",
-    "yaml": "^2.2.2"
+    "yaml": "^2.2.2",
+    "zod": "^3.22.4"
```

Comment:
I thought you said you had to remove `zod`

Reply:
Yes, my bad